website/docs/r/cloud_provider_snapshot_backup_policy.html.markdown

```diff
@@ -73,6 +73,47 @@ resource "mongodbatlas_cloud_provider_snapshot_backup_policy" "test" {
 }
 ```
 
+## Example Usage implementing at least one Policy Item
+
+```hcl
+resource "mongodbatlas_cluster" "my_cluster" {
```

Comment:
I think a comment here explaining "this cluster has already been created and is here as an example" would help.